versioned_docs/version-0.11.11/connectors/catalog/outbound/duckdb.mdx

```diff
@@ -51,7 +51,7 @@ meta:
   name: duckdb-connector
```

Comment:
There has been a lot of time without a release of this connector, so not sure if we should keep it. Fluvio client in the released connector should not be compatible with current fluvio version